include/itkMorphologicalContourInterpolator.h

```diff
@@ -60,7 +60,9 @@ class MorphologicalContourInterpolator :public ImageToImageFilter < TImage, TIma
   MorphologicalContourInterpolator()
     :m_Label(0),
     m_Axis(-1)
-  { }
+  {
+    m_LabeledSlices.resize(TImage::ImageDimension); //initialize with empty sets
```

Comment:
Please move the contructor to the .hxx file.  m_LabeledSlices can be set up in the initializer list, too.